corehq/apps/app_manager/models.py

```diff
@@ -3454,6 +3454,9 @@ def _copy_form(self, from_module, form, to_module):
         if 'unique_id' in copy_source:
             del copy_source['unique_id']
 
+        for lang, name in copy_source['name'].iteritems():
+            with override(lang):
+                copy_source['name'][lang] = _('Copy of %(name)s') % { 'name': name }
```

Comment:
If the string is short and/or there's only one var to sub in, you can often just do this:
```python
_('Copy of %s') % name
```
Also, while both are valid (and I normally wouldn't even comment on it except that you're somewhat new to python), the more modern form of string formatting is using the string's `format` method:
```python
_("Copy of {name}").format(name=name)
# or
_("Copy of {}").format(name)
```
For more, and for format options like alignment/padding or number representation, check out [the docs on string formatting](https://docs.python.org/2/library/string.html#formatspec)